rednotebook/gui/options.py

```diff
@@ -299,6 +318,8 @@ def check_version_action(widget):
                 _('Date format'), 'exportDateFormat',
                 tooltip=_('Used for dates in titlebar and exports.')
                 ),
+            IntegerOption(_('Tag display limit'), 'tagDisplayLimit',
```

Comment:
Let's use 'Tags in cloud' for the label and 'cloudMaxTags' for the config key.

Reply:
OK